front-end/src/util/formatBnBalance.ts

```diff
@@ -45,7 +45,7 @@ export  default function (value: BN | string, options: Options): string {
 		prefix = prefix.replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
```

Comment:
I'd be in favour of using a comma as a delimiter rather than a space. It's sort of the standard way of displaying balances.

Reply:
I'm strongly against that. It's standard in the US maybe, not in Europe.
Well Wikipedia seems to be ok with that: https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping

> Since 2003, the use of spaces as separators has been officially endorsed by SI/ISO 31-0 standard,[27] as well as by the International Bureau of Weights and Measures and the International Union of Pure and Applied Chemistry (IUPAC),[28][29] the American Medical Association's widely followed AMA Manual of Style, and the Metrication Board, among others.

This sums up my position, comma or points are misleading, and I still struggle with apps. I'd be even more in favor of a halfspace, or 0.3 rem, but this would require a little more work.
> Because of the confusion that could result in international documents, in recent years the use of spaces as separators has been advocated

Reply:
Ok, this discussion will have to move to a general design consistency discussion. Just checked a range of crypto apps out there and pretty much everybody is using commas.